src/CommunityToolkit.Aspire.Hosting.MassTransit.RabbitMQ/CommunityToolkit.Aspire.Hosting.MassTransit.RabbitMQ.csproj

```
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+      <AdditionalPackageTags>Hosting Aspire MassTransit RabbitMQ</AdditionalPackageTags>
```

```suggestion
      <AdditionalPackageTags>Hosting MassTransit RabbitMQ</AdditionalPackageTags>
```

Aspire isn't required as a tag